lib/consumer.js

```diff
@@ -112,7 +112,7 @@ Consumer.prototype.init = function () {
             return self.emit('error', err);
         }
 
-        if (self.options.fromOffset)
+        if (!self.options.fromOffset)
```

Comment:
@OndroNR If you look at the rest of the (old) code when the fromOffsets is true, then the messages are fetched from kafka with offset = 0 and the function is exited with the return statement `return self.fetch();`